pkg/private/build_zip.py

```diff
@@ -70,24 +125,13 @@ def main(args):
   if args.mode:
     default_mode = int(args.mode, 8)
 
-  with zipfile.ZipFile(args.output, 'w') as zip_file:
-    for f in args.files or []:
-      (src_path, dst_path) = helpers.SplitNameValuePairAtSeparator(f, '=')
-
-      dst_path = _combine_paths(args.directory, dst_path)
-
-      entry_info = zipfile.ZipInfo(filename=dst_path, date_time=ts)
-
-      if default_mode:
-        entry_info.external_attr = default_mode << 16
-
-      entry_info.compress_type = zipfile.ZIP_DEFLATED
+  with zipfile.ZipFile(args.output, mode='w') as zip_file:
```

Comment:
I've been writing all of my Python with 4-space indents, and much of the other python code is using 2-space indents.  Should we standardize on one or the other?

Reply:
Years of Google 2-space indent vs. other people's 4 space have left me ill-suited to choose. This code all started out as 2-space, and didn't change when we released it.
I'm fine with picking 4 space, but first let's pick a linter that warns of the indent and
then do the change in one big pass that has no functional changes.
Actually, maybe two passes. One to re-indent but not change line lengths, so we can compare before and after with diff -R -w and see that they are identical. And a second pass to allow the linter to rewrap the, now longer, lines.